api/envoy/api/v2/listener/listener.proto

```diff
@@ -188,6 +188,11 @@ message FilterChain {
 
   // See :ref:`base.TransportSocket<envoy_api_msg_core.TransportSocket>` description.
   core.TransportSocket transport_socket = 6;
+
+  // The unique name (or empty) by which this filter chain is known. If no name is provided,
+  // Envoy will allocate an internal UUID for the listener. If the listener is to be dynamically
+  // updated or removed via FCDS(not implemented yet) a unique name must be provided.
```

Comment:
```suggestion
  // updated or removed via FCDS a unique name must be provided.
```

Please mark this field not-implemented-hide until it's actually implemented.

Reply:
+1 on this